db/types/__init__.py

```diff
@@ -8,6 +8,7 @@
     # ischema_names supported by a PostgreSQL engine
     PostgresType.DECIMAL.value: sa_decimal,
     PostgresType.INTERVAL.value: datetime.Interval,
+    PostgresType.MATHESAR_CHAR.value: '"char"',
```

Comment:
@mathemancer @kgodey @dmos62 I have a question that I have to keep PostgresType.MATHESAR_CHAR.value as '"char"'  but I am receiving the typo error so can you please help me out of it?

Reply:
This dictionary _uses_ the value set in the `PostgresType` Enum as a key. The value at this point should be the python class, not the string.